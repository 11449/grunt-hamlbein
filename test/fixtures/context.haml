%h1= title
%ul
  - collection.each do |item|
    %li= item
%h2= hash['name']
%p= hash['description']
