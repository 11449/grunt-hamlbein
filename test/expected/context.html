<h1>This is just a test</h1>
<ul>
  <li>test1</li>
  <li>test2</li>
  <li>test3</li>
</ul>
<h2>Testicus</h2>
<p>Lorem ipsum dolor sit amet.</p>
